gulpfile.js

```diff
@@ -9,7 +9,7 @@ var gulp = require('gulp'),
 gulp.task('build-theme', function () {
     return gulp.src(['scss/bootstrap-dark.scss'])
```

Comment:
Assuming that we name the files `toggle-bootstrap.scss` and `toggle-bootstrap-dark.scss`, to have gulp generate the css files, you'd need to add `'scss/toggle-*.scss'` to the array, leading to `return gulp.src(['scss/bootstrap-dark.scss', 'scss/toggle-*.scss'])`.

This will allow someone to load `bootstrap-dark.css` or `toggle-bootstrap-dark.css` without importing the SCSS file.